angr/analyses/cfg/indirect_jump_resolvers/jumptable.py

```diff
@@ -629,8 +665,11 @@ def _parse_load_statement(load_stmt, state):
             # LoadG comes with a guard. We should apply this guard to the load expression
             guard_tmp = load_stmt.guard.tmp
             guard = state.scratch.temps[guard_tmp] != 0
-            jump_addr = state.memory._apply_condition_to_symbolic_addr(jump_addr, guard)
-
+            try:
+                jump_addr = state.memory._apply_condition_to_symbolic_addr(jump_addr, guard)
+            except: # pylint: disable=bare-except
+                l.exception("Error computing jump table address!")
+                return None
```

Comment:
Do not catch all possible exceptions. Please be specific about the exceptions you want to catch.

Reply:
I'm a bit conflicted on what to do here; there are _tons_ of possible exceptions thrown here, as that eventually calls into angr/claripy/etc, and the answer to all of them is the same: give up. This is infinitely preferable, in my opinion, to people complaining that Jumptable crashed their CFG.  Maybe we can move this out to IndirectJumpResolver or whatever (e.g., no resolver should crash CFG, but should prefer giving up with a Logger.exception() call instead)

Reply:
All fixed, as we discussed on Slack